### ISSUE TITLE
Failed to run the image with the default port mapping

### ISSUE THREAD

**Issue by jrathnayake** [wso2/docker-apim#226]
**Description:**
I tried to run the APIM docker image following below URL:
https://wso2.com/api-management/install/docker/community/get-started/

When I try to start the container using the same command - 

docker run -it -p 9443:9443 -p 8243:8243 -p 8280:8280 wso2/wso2am

I get the below error:
docker: Error response from daemon: driver failed programming external connectivity on endpoint recursing_kowalevski (81f29976334673d40c029887537708b7c8ac94b2fdf601919de7abf31d2a62d6): Error starting userland proxy: listen tcp 0.0.0.0:9443: bind: address already in use.
![docker](https://user-images.githubusercontent.com/45289659/61140781-bd31bf00-a4e9-11e9-8dcd-8d163595f37a.PNG)
![error](https://user-images.githubusercontent.com/45289659/61140787-c02caf80-a4e9-11e9-88a7-fb65860aafa1.PNG)

I have attached the current configuration of the Docker and the error message.

I tried the same with a MacOS and I get the same behavior.

I tried to change the port mapping using below command and it worked:
docker run -it -p 9449:9443 -p 8243:8243 -p 8280:8280 wso2/wso2am

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by robertlagrant** [comment]
@jrathnayake I don't understand the issue. Didn't you just have something already running on the default port?

**Comment by jrathnayake** [comment]
No, I didn't have anything running on the same port. To make sure, I tried this in 2 different laptops - Windows and Mac.

**Comment by asafg84** [comment]
ping

**Comment by vsobin01** [comment]
I have the same problem. 9443 was free.

ERROR: for api-manager  Cannot start service api-manager: driver failed programming external connectivity on endpoint docker_api-manager_1 (b4757657226c7d1fb96f1d87c4ec55a5f17a5a118c9e0dd59bf472a61e57cd44): Error starting userland proxy: **listen tcp 0.0.0.0:9443: bind: address already in use**
ERROR: Encountered errors while bringing up the project.

$ docker-compose ps
        Name                      Command                State                          Ports
-------------------------------------------------------------------------------------------------------------------
broker                 /etc/confluent/docker/run        Up         0.0.0.0:29092->29092/tcp, 0.0.0.0:9092->9092/tcp
docker_api-manager_1   /home/wso2carbon/docker-en ...   Exit 128
kafdrop                /kafdrop.sh                      Up         0.0.0.0:9000->9000/tcp
zookeeper              /etc/confluent/docker/run        Up         0.0.0.0:2181->2181/tcp, 2888/tcp, 3888/tcp

**Comment by oscarvesga** [comment]
> No, I didn't have anything running on the same port. To make sure, I tried this in 2 different laptops - Windows and Mac.

Did you use `docker ps -a` to be sure?

**Comment by vsobin01** [comment]
The problem was with k8s. After I deactivated k8s, everything works fine -> MacOS.
